plugins/plugin-docker/che-plugin-openshift-client/src/main/java/org/eclipse/che/plugin/openshift/client/OpenShiftConnector.java

```diff
@@ -1428,4 +1432,38 @@ private boolean isTerminalAgentInjected(final Set<String> exposedPorts) {
     private boolean isDevMachine(final Set<String> exposedPorts) {
         return exposedPorts.contains(CHE_WORKSPACE_AGENT_PORT + "/tcp");
     }
+
+    @Override
+    public SystemInfo getSystemInfo() throws IOException {
+        PodList chePods = openShiftClient.pods().inNamespace(this.openShiftCheProjectName)
+                .withLabel(OPENSHIFT_DEPLOYMENT_CONFIG_LABEL, OPENSHIFT_DEPLOYMENT_CONFIG_VALUE).list();
```

Comment:
Is this step necessary? I feel like explicitly checking for `deploymentconfig=che` is prone to error if the deploy script is changed. Is it sufficient to just do `openShiftClient.pods().inNamespace(...).list()`?